src/vcpkg/base/system.cpp

```diff
@@ -70,6 +70,15 @@ namespace vcpkg
     CPUArchitecture get_host_processor()
     {
 #if defined(_WIN32)
+        auto raw_identifier = get_environment_variable("PROCESSOR_IDENTIFIER");
+        if (const auto id = raw_identifier.get())
+        {
+            if (Strings::contains(id, "ARMv") && Strings::contains(id, "(64 bit)"))
```

Comment:
```suggestion
            if (Strings::contains(id, StringLiteral{"ARMv"}) && Strings::contains(id, StringLiteral{"(64 bit)"}))
```

will fix your build failure